kompendium-auth/build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+plugins {
+  `java-library`
+  `maven-publish`
+}
+
+dependencies {
+  implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
+  implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+  implementation(libs.bundles.ktor)
+  implementation("io.ktor:ktor-auth:1.5.3")
+  implementation("io.ktor:ktor-auth-jwt:1.5.3")
```

Comment:
so we can keep the gradle build shiny ✨can you add these two under the `gradle/libs.versions.toml` as a `ktor-auth` bundle? should be something like 

```toml
[Libraries]
ktor-auth-lib = { group = "io.ktor", name = "ktor-auth", version.ref = "ktor" }
ktor-auth-jwt= { group = "io.ktor", name = "ktor-auth-jwt", version.ref = "ktor" }
[Bundles]
ktor-auth = [ "ktor-auth-lib", "ktor-auth-jwt" ] 
```

Then you should just be able to reference it as `implementation(libs.bundles.ktor.auth)`

Reply:
aah cool, I didn't know this was possible. I've updated it as above except the bundle name is ktorAuth. I got errors when adding `-` or `.` in the bundle name.

Reply:
ah yep your totally right I always forget if it is `my-bundle` or `myBundle` 😅